rel/i18n/emqx_bridge_hstreamdb.hocon

```diff
@@ -47,4 +47,58 @@ NOTE: When you use `raw record` template (which means the data is not a valid JS
 record_template.label:
 """HStream Record"""
 
+action_parameters.desc:
+"""Action specific configuration."""
+
+action_parameters.label:
+"""Action"""
+
+grpc_flush_timeout.desc:
+"""Period for flushing gRPC calls to the HStreamDB server"""
+
+grpc_flush_timeout.label:
+"""gRPC Flush Period"""
+
+aggregation_pool_size.desc:
+"""Size of Record Aggregation Pool"""
```

Comment:
```suggestion
"""Size of record aggregation pool."""
```

Reply:
might need more information. e.g. what does it mean to set a larger pool or smaller pool.